doc/live_iso.md

```diff
@@ -49,3 +70,158 @@ Notes:
+
+  The disadvantage is that the hashed password is quite long and is not easy to
+  type it into the boot prompt manually. It makes sense in environments where
+  you can prepare the boot parameters in advance like in PXE boot or some
+  virtual machines.
+
+  See more details about creating a hashed password [below](
+  #creating-a-hashed-password).
+
+### Interactive Input
+
+You can enter your password during boot in an interactive session. Again, there
+are two options:
+
+* Use `agama.password_dialog` boot option to start and interactive dialog during
+  the boot process. This uses a nice dialog for entering and confirming the
+  password. However, in some situations the full screen dialog might not be
+  displayed correctly or some messages might be displayed over it. In that case
+  you might use the `Ctrl+L` key shortcut to refresh the screen. If it still
+  does not work then try using the other option below.
+
+* Use `agama.password_systemd` boot option to ask for the password in a simple
+  prompt. This is similar to the option above, but the advantage is that this
+  solution does not use a full screen dialog but a single line prompt so it
+  should work better in special environments like a serial console.
+
+The Agama and the SSH server are not started until a password is configured.
+This avoid using the default password from the medium accidentally.
+
+### Injecting the Default Password Into the ISO Image
+
+Another option is to inject your custom hashed password directly into the ISO
+image. The advantage is than you can easily use the same image for installing
+multiple machines and you do not need to configure anything during the boot.
+
+To inject a new password into the ISO run:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia --add-tag "agama_password=$((openssl passwd -6) | base64 -w 0)" agama.iso
+```
+
+It will interactively ask for a password then it will be hashed using the SHA512
+algorithm, encoded to the Base64 encoding and appended to the application area
+in the ISO file. If you want to update the password then just the same command
+again, it will overwrite the existing password.
+
+See the [Creating a Hashed Password](#creating-a-hashed-password) section below
+if you want to use a different hashing algorithm than SHA512.
+
+To check all tags present in an ISO file use this command:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia agama.iso
+```
+
+If you want to remove the password setting from the ISO image then run:
+
+```sh
+# replace the agama.iso name with your image name
+tagmedia --remove-tag agama_password agama.iso
+```
+
+:information_source: *Note: The image usually already contains some other tags,
+like the checksums for verifying the medium integrity. Do not touch them!*
+
+### Random Password as a Fallback
+
+When no password is specified or the entering the password interactively was
```

Comment:
"or the entering the password" sounds weird. I guess there is a left-over "the".